main.js

```diff
@@ -26,136 +26,99 @@ categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
 
 //Add functions below
+function show(element) {
+  element.classList.remove('hidden');
+}
+
+function hide(element) {
+  element.classList.add('hidden');
+}
+
 function submitForm() {
   event.preventDefault();
   if (checkInputValidity()) {
     createNewActivity();
-  };
-};
+    hide(activityInputForm);
```

Comment:
Nice!! We did lose this during that merge b/c we replaced it with refactored code.